### PR TITLE
QA - New wallet key line instructions for example section

### PR DESCRIPTION
**Pull request by obmatter**
# What :computer: 
* First thing updated with this PR
* Second thing updated with this PR
* Third thing updated with this PR

# Why :hand:
* Reason why first thing was added to PR
* Reason why second thing was added to PR
* Reason why third thing was added to PR

# Evidence :camera:
Include screenshots, screen recordings, or `console` output here demonstrating that your changes work as intended

# Notes :memo:
* Any notes/thoughts that the reviewers should know prior to reviewing the code?

### CODE REVIEW

File: docs/dev/how-to/send-message-l2-l1.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ yarn add zksync-web3 ethers@5 typescript @types/node ts-node
 
 5. Create a `file.ts` file in the root directory with the next script:
 
+Note: Replace `<YOUR_PRIVATE_KEY>` with private key of the wallet to use.
```
Comment (MexicanAce) [inline docs/dev/how-to/send-message-l2-l1.md:71]: We should not recommend users type their Private Key in scripts. We should update this script to use the `.env` file to better ensure they don't accidentally share them or commit them into git

**Review by idea404 — CHANGES_REQUESTED** [review]
Agree with @MexicanAce comment on adjusting text to exemplify using `.env` instead of hardcoding private information

File: docs/dev/how-to/send-message-l2-l1.md (hunk 1 above)
Comment (obmatter) [reply docs/dev/how-to/send-message-l2-l1.md]: @MexicanAce fixed by adding instructions for .env file and also updating the scipt
image

Comment (obmatter) [reply docs/dev/how-to/send-message-l2-l1.md]: (For step 5 used text from another tutorials to remain docs look solid)

**Comment by obmatter** [comment]
@MexicanAce @idea404 can you please review the PR with the already implemented fix for hardcoded wallet?
Thank you in advance!

**Review by idea404 — APPROVED** [review]

**Pull request merged**